flang/lib/Optimizer/HLFIR/Transforms/OptimizedBufferization.cpp

```diff
@@ -988,8 +988,15 @@ class ReductionConversion : public mlir::OpRewritePattern<Op> {
           op, "Currently minloc/maxloc is not handled");
     } else if constexpr (std::is_same_v<Op, hlfir::MaxvalOp> ||
                          std::is_same_v<Op, hlfir::MinvalOp>) {
+      auto ty = op.getType();
```

Comment:
In flang we spell out the type if it isn't obvious from the context (e.g. written as part of the rhs expression)
```suggestion
      mlir::Type ty = op.getType();
```

Reply:
Done 👍